test/scripts/utils.js

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs'
+import path from 'path'
+import { Miniflare } from 'miniflare'
+
+import { createR2Bucket } from './r2.js'
+import { globals } from './worker-globals.js'
+
+export function getMiniflare () {
+  let envPath = path.join(process.cwd(), '../../.env')
+  if (!fs.statSync(envPath, { throwIfNoEntry: false })) {
+    // @ts-ignore
+    envPath = true
+  }
+
+  return new Miniflare({
+    envPath,
+    scriptPath: 'dist/worker.mjs',
+    port: 8788,
+    packagePath: true,
+    wranglerConfigPath: true,
+    // We don't want to rebuild our worker for each test, we're already doing
+    // it once before we run all tests in package.json, so disable it here.
+    // This will override the option in wrangler.toml.
+    // buildCommand: undefined,
```

Comment:
the way that i do it, i just get miniflare isolation per test file because ava run each test file in different worker threads. 
but its nice because watch mode reloads the worker and not just the tests

Reply:
reverted